src/mbgl/map/map.cpp

```diff
@@ -997,6 +1004,22 @@ void Map::Impl::onUpdate(Update flags) {
     
 void Map::Impl::onStyleLoaded() {
     view.notifyMapChange(MapChangeDidFinishLoadingStyle);
+    
+    if (initialCameraOptions == nullptr) {
+        // user manipulated camera before style was loaded
+        return;
+    }
+    
+    // move camera to default camera of style
+    initialCameraOptions->center = style->getDefaultLatLng();
+    initialCameraOptions->zoom = style->getDefaultZoom();
+    initialCameraOptions->angle = style->getDefaultBearing();
```

Comment:
In the style specification, the [`bearing`](https://www.mapbox.com/mapbox-gl-style-spec/#root-bearing) property is measured in degrees clockwise from north. However, `CameraOptions::angle` is measured in radians counterclockwise from north. We need to convert either here or in `mbgl::Style::setJSON()`, where we set the default bearing based on the value parsed out of the style JSON. We should also document the units and direction of `getDefaultBearing()`.